index.js

```diff
@@ -863,6 +884,13 @@
     return f(z, this.value);
   });
 
+  //  Just#sequence :: Applicative f => Maybe (f a) ~> (b -> f b) -> f (Maybe a)
+  Just.prototype.sequence =
+    def('Just#sequence', [Function], function(of) {
+      void of;
```

Comment:
Okay. I wouldn't have minded keeping it in that case. I assumed we were appeasing some overeager linter. :)